frb_example/pure_dart/dart/lib/main.dart

```diff
@@ -323,8 +323,8 @@ void main(List<String> args) async {
 
   test('dart call getMessage()', () async {
     var message = await api.getMessage();
-    expect(message is RenderPixel, true);
-    message as RenderPixel;
+    expect(message is ApplicationMessage_RenderPixel_Variant, true);
+    message as ApplicationMessage_RenderPixel_Variant;
```

Comment:
Please add some (very simple) tests calling the new functions that you have generated :)

Since we want to ensure names are correct, we may need:

```
expect(TheClassThatYouAreInterested.toString(), 'TheClassThatYouAreInterested');

Reply:
Ok I made it up from previous tests in 5fe576edf13ec5c187ca3ee0041c21c253849d5d

Reply:
Tests like 

`expect(Speed_GPS_Variant.toString(), 'Speed_GPS_Variant');`

may be helpful to ensure the names are what we expect

Reply:
> expect(Speed_GPS_Variant.toString(), 'Speed_GPS_Variant');

I haven't committed yet since the test doesn't pass.
It fails with :
```dart
expect(Speed_GPS_Variant.toString(), 'Speed_GPS_Variant'); // ❌
// The method 'toString' isn't defined for the type 'Speed_GPS_Variant'.
// Try correcting the name to the name of an existing method, or defining a method named 'toString'.
```
But it passes with :
```dart
expect(Speed_Unknown_Variant().toString(), 'Speed.unknown()'); // ✅
expect(Speed_GPS_Variant(0.0).toString(), 'Speed.gps(field0: 0.0)'); // ✅
```

Reply:
This works:

```dart
(Speed_GPS_Variant as Type).toString()

Reply:
done in 4ada517f9f2db55219e9156c94a6d7e01bd999cb

p.s: I removed `as Type`
<details><summary>because Dart warns about it</summary>

<img width="565" alt="Screen Shot 2565-08-02 at 10 51 52" src="https://user-images.githubusercontent.com/21016014/182288375-85ef6456-1c65-4a70-a73b-a932f0092ccc.png">

</details>